graftm/hmmer.py

```diff
@@ -925,7 +960,7 @@ def search_and_extract_nucleotides_matching_nucleotide_database(self,
         return result
         
     @T.timeit
-    def align(self, input_path, output_path, directions):
+    def align(self, input_path, output_path, directions, pipeline):
```

Comment:
good call


________________________________
From: Ben J Woodcroft <notifications@github.com>
Sent: Friday, 21 August 2015 1:22 PM
To: geronimp/graftM
Cc: Mr Joel Boyd
Subject: Re: [graftM] Geronimp diamond issue (#97)


In graftm/hmmer.py<https://github.com/geronimp/graftM/pull/97#discussion_r37603716>:

> @@ -925,7 +960,7 @@ def search_and_extract_nucleotides_matching_nucleotide_database(self,
>          return result
>
>      @T.timeit
> -    def align(self, input_path, output_path, directions):
> +    def align(self, input_path, output_path, directions, pipeline):


hey can you add comment for the new parameter?

-
Reply to this email directly or view it on GitHub<https://github.com/geronimp/graftM/pull/97/files#r37603716>.